frontend/src/pages/identityVerification/apis/postIdentityVerification.ts

```diff
@@ -0,0 +1,13 @@
+import { apiClient } from '../../../common/apis/apiClient';
+import { API_ENDPOINTS } from '../../../common/constants/apiEndpoints';
+import { IdentityVerificationInfo } from '../components/types/IdentityVerificationInfo';
```

Comment:
p3: import와 { 사이에 type이 빠졌어!

```suggestion
import type { IdentityVerificationInfo } from '../components/types/IdentityVerificationInfo';
```

eslint 적용이 안된 것 같아! 😅 

frontend 파일로 들어가서 lint 적용 스크립트를 실행시켜보면 좋을 것 같아 👍